### ISSUE TITLE
Update to work with latest twitter.com redesign

### ISSUE THREAD

**Issue by ElijahLynn** [tannerhodges/twitter-print-styles#2]
When I "print" nothing different appears. Should this be working or does it need an update?

**Comment by tannerhodges** [comment]
Hey Elijah, sorry. I haven't updated it since Twitter's redesign.

**Comment by ElijahLynn** [comment]
Thanks for confirming. In the COVID era, I've been printing a lot more tweets for a scrapbook. If I get some time I'll try to make a PR to get it working with the latest redesign.

**Comment by tannerhodges** [comment]
🙏 That'd be awesome. Thanks.

**Comment by millou54** [comment]
Please, please, update Twitter Print Styles to work with latest twitter.com redesign. I haven't come across any alternative that works remotely as well as TPS (and I've looked hard). Ready to help if I can.

**Comment by tannerhodges** [comment]
Hey all, I finally hunkered down and got this thing up and running again—at least enough to use with the latest version of Twitter.

## Pending Review

Just submitted the new version for review in the Chrome Web Store.

I'll post here & publish a new release once it's officially live.

![Example of Twitter Print Styles v2](https://user-images.githubusercontent.com/1619485/91786490-bb3b2b80-ebd5-11ea-9671-73b84425a2ff.gif)

## How to Test

If you want to test the changes while we wait on review, you can check out https://github.com/tannerhodges/chrome-twitter-print-styles/pull/5 and follow these steps:

1. Download the ZIP: [chrome-twitter-print-styles-v2.zip](https://github.com/tannerhodges/chrome-twitter-print-styles/files/5153538/chrome-twitter-print-styles-v2.zip)
2. Unzip the folder.
3. Go to `chrome://extensions/`.
4. Click **Load unpacked** and select the unzipped folder.
5. Confirm the extension shows up under **Extensions** in the top right of your browser.
6. Go to twitter.com and test it on a few tweets.

If you run into any issues, comment in the PR (#5) to let me know.

Again, I'll follow up here once the update is officially published in the Chrome Web Store.

🙏 Thanks all for your patience.

**Comment by millou54** [comment]
Gorgeous! I got it to work on Chrome and Brave easily. Thanks tons.
One minor request: when you go for save rather than print, it generates a file name that can be very long and exceed the max file name length of the OS (255 on MacOS, no idea on Windows). Can you limit the generated file name to the first m characters (m < 255 or whatever is reasonable)?

**Comment by tannerhodges** [comment]
## 🎉 v2 Published

Wow, that was faster than I expected.

The update is officially live on the Chrome Web Store:

https://chrome.google.com/webstore/detail/twitter-print-styles/bepilablapiogeghmjiopiaoikgdcgjo

## 📝 Max Filename Size

> Can you limit the generated file name to the first m characters (m < 255 or whatever is reasonable)?

👌 Definitely.

Since the v2 release is already live, I've opened a separate issue to fix the filename and release that as v2.0.1.

👉 https://github.com/tannerhodges/chrome-twitter-print-styles/issues/6 I'll follow up in both issues once that fix is released (probably sometime over Labor Day weekend).

**Comment by tannerhodges** [comment]
@millou54 I've added the max filename update with a handful of other fixes in https://github.com/tannerhodges/chrome-twitter-print-styles/pull/8.

Would you mind testing the changes to see if that fixes the filename issue? (Notes + ZIP to test are in the PR description.)
